### ISSUE TITLE
settings editor ignores $HOME/.gitconfig on Windows

### ISSUE THREAD

**Issue by af4jm** [gitextensions/gitextensions#6707]
## Current behaviour

I have ~/.gitconfig with most global settings and ~/.config/git/config with some machine-specific settings (i.e. ~/.gitconfig is identical on all machines I use, ~/config/git/config has per-machine overrides)

User & email are defined in ~/.gitconfig, but GitExtensions claims they are not set.  If I set them in the UI, they're written to ~/.config/git/config (along with several other settings which are already in ~/.gitconfig)

## Expected behaviour

GitExtensions recognizes that the user & email are set in ~/.gitconfig

## Steps to reproduce

Make sure ~/.gitconfig has user & email and ~/.config/git/config doesn't, then open GitExtensions > Settings

## Did this work in previous version of GitExtensions

unknown (I'm just looking for Windows Explorer icon overlays and figured out that this tools doesn't include them--PowerShell is the only git UI I ever use)

## Environment

- Git Extensions 3.1.0.5877
- Build bb1687f6bd0450d0c1a2bae10a37c65493fc1294
- Git 2.21.0.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.8.3761.0

## Diagnostics

Noticed in the Command log...
git config --get ą
yes that's an "a" with some sort of accent, and it blows up at the command line
git config --list
gives the combination of both files (when run outside of a git repo)

**Comment by gerhardol** [comment]
GE uses the the HOME set in settings:

![image](https://user-images.githubusercontent.com/6248932/58646183-2b893c00-8305-11e9-8550-ead63b2f4040.png)

**Comment by af4jm** [comment]
Yes @gerhardol , and in my case $HOME = $USERPROFILE... yet GitExtensions ignores $HOME\.gitconfig and only looks at $HOME/.config/git/config

per documentation (https://git-scm.com/docs/git-config --global), writes should always go to .gitconfig if it exists or if neither file exists,
and should go to .config/git/config if and only if that file exists and .gitconfig does not exist
